*** gd.h	Thu Jan  8 17:50:44 1998
--- gd.h.orig	Thu Jan  8 17:52:43 1998
***************
*** 100,106 ****
  void gdImageFilledRectangle(gdImagePtr im, int x1, int y1, int x2, int y2, int color);
  int gdImageBoundsSafe(gdImagePtr im, int x, int y);
  void gdImageChar(gdImagePtr im, gdFontPtr f, int x, int y, int c, int color);
! void gdImageCharUp(gdImagePtr im, gdFontPtr f, int x, int y, int c, int color);
  void gdImageString(gdImagePtr im, gdFontPtr f, int x, int y, char *s, int color);
  void gdImageStringUp(gdImagePtr im, gdFontPtr f, int x, int y, char *s, int color);
  
--- 100,106 ----
  void gdImageFilledRectangle(gdImagePtr im, int x1, int y1, int x2, int y2, int color);
  int gdImageBoundsSafe(gdImagePtr im, int x, int y);
  void gdImageChar(gdImagePtr im, gdFontPtr f, int x, int y, int c, int color);
! void gdImageCharUp(gdImagePtr im, gdFontPtr f, int x, int y, char c, int color);
  void gdImageString(gdImagePtr im, gdFontPtr f, int x, int y, char *s, int color);
  void gdImageStringUp(gdImagePtr im, gdFontPtr f, int x, int y, char *s, int color);
  
