*** libgd.c	Thu Jan  8 17:50:44 1998
--- libgd.c.orig	Thu Jan  8 17:52:37 1998
***************
*** 450,456 ****
  	}
  }
  
! void gdImageCharUp(gdImagePtr im, gdFontPtr f, int x, int y, int c, int color)
  {
  	int cx, cy;
  	int px, py;
--- 450,456 ----
  	}
  }
  
! void gdImageCharUp(gdImagePtr im, gdFontPtr f, int x, int y, char c, int color)
  {
  	int cx, cy;
  	int px, py;
***************
*** 479,485 ****
  	int l;
  	l = strlen(s);
  	for (i=0; (i<l); i++) {
! 		gdImageChar(im, f, x, y, (unsigned char)s[i], color);
  		x += f->w;
  	}
  }
--- 479,485 ----
  	int l;
  	l = strlen(s);
  	for (i=0; (i<l); i++) {
! 		gdImageChar(im, f, x, y, s[i], color);
  		x += f->w;
  	}
  }
***************
*** 490,496 ****
  	int l;
  	l = strlen(s);
  	for (i=0; (i<l); i++) {
! 		gdImageCharUp(im, f, x, y, (unsigned char)s[i], color);
  		y -= f->w;
  	}
  }
--- 490,496 ----
  	int l;
  	l = strlen(s);
  	for (i=0; (i<l); i++) {
! 		gdImageCharUp(im, f, x, y, s[i], color);
  		y -= f->w;
  	}
  }
